share/dkg/pedersen/dkg.go

```diff
@@ -61,38 +62,22 @@ type Config struct {
 	// OldNodes field.
 	Share *DistKeyShare
 
-	// New threshold to use if set. Default will be returned by `vss.MinimumT()`
-	Threshold int
```

Comment:
What values do you use as `NewThreshold` and `OldThreshold`: since I guess you require all the nodes present anyway, I can try to make a small failing test...